_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,137 @@
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to neovim, visit [the github
+page](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md) for details.
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](gitter.im/neovim) (also
+accessible via [IRC](https://webchat.freenode.net/?channels=neovim)), or visit
+the [neovim subreddit](https://www.reddit.com/r/neovim/), which many of the
+developers read. Taking part in conversations is a contribution, too!
+
+
+What's new
+----------
+
+The [last newsletter](https://neovim.io/news/2016/11/) has gathered some dust,
+so we want to bring you up-to-date in the neovim world. The elephant in the room
+is certainly
+
+### Release 0.2
+
+It features windows support, various terminal improvements, full API
+documentation for developers and a host of smaller fixes and improvements. A small highlight: A long standing issue with using `<Ctrl-h>` has been [fixed](https://github.com/neovim/neovim/wiki/FAQ#my-ctrl-h-mapping-doesnt-work).
+
+See
+the [release page](https://github.com/neovim/neovim/releases/tag/v0.2.0) for a
+detailed summary and download opportunities.
+
+### 0.2.1-dev for Users
+- **@bfredl** brought us window specific background color (https://github.com/neovim/neovim/pull/6597)
+- Even more terminal improvements were contributed by **@jdebp** (https://github.com/neovim/neovim/pull/6816)
+- A feature request as old as ~time~ ~vim~(well, very old for sure) itself: Cursorline and cursorcolumn now respect syntax highlighting. Thanks **@zhou13**! (https://github.com/neovim/neovim/pull/6380)
+- From **@ZyX-l** comes a way to color the command line (https://github.com/neovim/neovim/pull/6364)
+- For windows users:
+  - Neovim can now be used on the console thanks to **@equalsraf** (https://github.com/neovim/neovim/pull/6315)
+  - **@erw7** implemented `:terminal` (https://github.com/neovim/neovim/pull/7007)
+  - **@aignas** improved path handling on windows (https://github.com/neovim/neovim/pull/7349)
+- For Linux users:
+  - **@AdnoC** contributed AppImage packages for Linux (https://github.com/neovim/neovim/pull/6638)
+
+
+
+### 0.2.1-dev for Developers
+- **@ZyX-l** enabled static Analysis with PVS (https://github.com/neovim/neovim/pull/6493)
+- Thanks to **@dzhou121**, external UIs can now draw the tabline (https://github.com/neovim/neovim/pull/6583)
+- Despite it being 100% positively impossible, **@ZyX-l** managed to refactor `eval.c` (https://github.com/neovim/neovim/pull/5119)
```

Comment:
I did not refactor the entire `eval.c`, I only moved a selected number of functions to another location and refactored them. Bram did more splitting.

Reply:
Ah ok. I left that point out then, hope that's ok for you.